generator/src/main.rs

```diff
@@ -1308,13 +1317,10 @@ impl Parser {
     /// Generate high-level code
     #[allow(clippy::cognitive_complexity)] // TODO
     fn generate_hl(&self) -> TokenStream {
-        const BLACKLISTED_COMMANDS: [&str; 3] = [
-            "xrNegotiateLoaderRuntimeInterface",
-            "xrNegotiateLoaderApiLayerInterface",
-            "xrCreateApiLayerInstance",
-        ];
```

Comment:
Keeping track of the commands listed in the feature tag for `XR_VERSION_1_0` allowed me to get rid of this! 😄

Reply:
Great catch!